Makefile

```diff
@@ -135,6 +135,10 @@ DEFHOTPLUGVFIOONROOTBUS := false
 SED = sed
 
 CLI_DIR = cli
+SHIMV2 = containerd-shim-kata-v1
```

Comment:
Hi @jodh-intel 
I had communicated with crosby.michael@gmail.com about the shim name style and he explained the reason as below: 

Because we want to version the shims separately from the runtime/manager in containerd.  The containerd implementation is at v2 but the runc shim is the first release so it's v1.  If we do a large change, we can update the runc shim to a v2 binary while we make sure that v1 shims continue to work and run the same.  You would be able to do this with kata as well without impacting existing containers on a v1 kata shim.

Reply:
@lifupan I think you may simply remove the `-v1` here. For the containerd side, the `-v1` means using the v1 implementation to support v2 API, which is not the case for us. On the contrary, we don't have a v1 shim, we have only this one.

Reply:
+1, we don't need to stick to "v1" sth, it's confusing